v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/loadtesting/DataStreamToSpannerBlobTable.java

```diff
@@ -32,7 +32,7 @@
 @RunWith(JUnit4.class)
 public class DataStreamToSpannerBlobTable extends DataStreamToSpannerLTBase {
```

Comment:
We should also add `LT` suffix in order to make load test action to pick up these classes.